nomad/structs/structs.go

```diff
@@ -5236,9 +5238,59 @@ func IsRecoverable(e error) bool {
 
 // ACLPolicy is used to represent an ACL policy
 type ACLPolicy struct {
-	Name  string // Unique name
-	Rules string // HCL or JSON format
+	Name        string // Unique name
```

Comment:
You may want to add a validate and ensure the name/description is length bounded